fem/eltrans.hpp

```diff
@@ -349,12 +370,38 @@ class IntegrationPointTransformation
    void Transform (const IntegrationRule  &, IntegrationRule  &);
 };
 
-class FaceElementTransformations
+class FaceElementTransformations : public IsoparametricTransformation
 {
+private:
+   int side;
+
 public:
    int Elem1No, Elem2No, FaceGeom;
```

Comment:
This is a good idea.  `geom` is a different type from `FaceGeom` which may lead to some warnings but perhaps this will highlight the change and encourage people to switch to the new interface.

I added a doxygen comment to mark this as deprecated since we don't yet have a more formal way to handle deprecated features.